### PR TITLE
[Browser Flags article] Add `about:blank` exception

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add `about:blank` exception. Thanks, @naskooskov!

### CODE REVIEW

File: src/site/content/en/blog/browser-flags/index.md [hunk 1]
```diff
@@ -78,6 +78,11 @@ URLs get rewritten to `chrome://`, in Edge to `edge://`, and so on for all vendo
 web thing together, and this is `about://` all of us! Whenever you see instructions that include the
 `about://` scheme, your Chromium browser of choice will do the right thing.
 
+{% Aside %}
+A notable exception from the rewriting mechanism is `about:blank` (without the `//`), which
```
Comment (tomayac) [inline src/site/content/en/blog/browser-flags/index.md:82]:
```suggestion
A notable exception from the rewriting mechanism is [`about:blank`](about:blank) (without the `//`), which
```

File: src/site/content/en/blog/browser-flags/index.md [hunk 2]
```diff
@@ -78,6 +78,11 @@ URLs get rewritten to `chrome://`, in Edge to `edge://`, and so on for all vendo
 web thing together, and this is `about://` all of us! Whenever you see instructions that include the
 `about://` scheme, your Chromium browser of choice will do the right thing.
 
+{% Aside %}
+A notable exception from the rewriting mechanism is [`about:blank`](about:blank) (without the `//`), which
+displays a blank, empty window.
```
Comment (tomayac) [inline src/site/content/en/blog/browser-flags/index.md:83]:
```suggestion
displays a blank, empty document.
```

File: src/site/content/en/blog/browser-flags/index.md [hunk 3]
```diff
@@ -78,6 +78,11 @@ URLs get rewritten to `chrome://`, in Edge to `edge://`, and so on for all vendo
 web thing together, and this is `about://` all of us! Whenever you see instructions that include the
 `about://` scheme, your Chromium browser of choice will do the right thing.
 
+{% Aside %}
+A notable exception from the rewriting mechanism is [`about:blank`](about:blank) (without the `//`), which
```
Comment (jpmedley) [inline src/site/content/en/blog/browser-flags/index.md:82]:
```suggestion
A notable exception to the rewriting mechanism is [`about:blank`](about:blank) (without the `//`), which
```

Comment (jpmedley) [reply src/site/content/en/blog/browser-flags/index.md:82]: Idiomatic English.

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Pull request merged**